.travis.yml

```diff
@@ -1,15 +1,16 @@
 sudo: false
+
 language: node_js
+
 node_js:
   - "0.10"
```

Comment:
Yea i think we never said "grunt drops 0.10 support" so we kept this, unless we really need to drop it.